mantidimaging/core/io/saver.py

```diff
@@ -57,7 +58,8 @@ def save(images: Images,
          zfill_len=DEFAULT_ZFILL_LENGTH,
          name_postfix=DEFAULT_NAME_POSTFIX,
          indices=None,
-         progress=None) -> Union[str, List[str]]:
+         progress=None,
+         pixel_depth=None) -> Union[str, List[str]]:
```

Comment:
Changing where progress is would break any places that already use it as an Arg and then pass what was previously passed to progress to the pixel_depth parameter. I am following best practice by adding onto the end of the function and using a default as it allows all previous calls to still be completely valid and working,  if I implemented this change it would cause previous calls to break.

Reply:
I will however add missing docs for this function good spot :+1:

Reply:
I'd like to keep `progress` as the last parameter across the repository. The only place that passes the `progress` param is in `main.model` and that uses a `kwarg`, shouldn't break anything else.

Reply:
I understand why you would like that but I dislike breaking best practice to achieve something that doesn't seem to have value (to me) for refactoring, maintainability and testability.

Reply:
On a potentially high-risk change I would agree, but on a single call-site it doesn't matter, especially when the call uses kwargs anyway so the order is irrelevant, this is purely for aesthetics at this point.